diff --git a/api/capi/include/nnstreamer.h b/api/capi/include/nnstreamer.h
index a18a066..ab1808c 100644
--- a/api/capi/include/nnstreamer.h
+++ b/api/capi/include/nnstreamer.h
@@ -28,7 +28,15 @@
 #include <stdbool.h>

 /* Apply modify_nnstreamer_h_for_nontizen.sh if you want to use in non-Tizen Linux machines */
-#include <tizen_error.h>
+#include <errno.h>
+#define TIZEN_ERROR_NONE (0)
+#define TIZEN_ERROR_INVALID_PARAMETER (-EINVAL)
+#define TIZEN_ERROR_STREAMS_PIPE (-ESTRPIPE)
+#define TIZEN_ERROR_TRY_AGAIN (-EAGAIN)
+#define TIZEN_ERROR_UNKNOWN (-1073741824LL)
+#define TIZEN_ERROR_TIMED_OUT (TIZEN_ERROR_UNKNOWN + 1)
+#define TIZEN_ERROR_NOT_SUPPORTED (TIZEN_ERROR_UNKNOWN + 2)
+#define TIZEN_ERROR_PERMISSION_DENIED (-EACCES)

 #ifdef __cplusplus
 extern "C" {
